kolibri/utils/options.py

```diff
@@ -32,6 +32,28 @@
             "envvars": ("KOLIBRI_DATABASE_HOST",),
         },
     },
+    "Server": {
```

Comment:
This new section name might sound generic, but it's IMO a good name, as opposed to `HTTP_SERVER`, because we would risk having maybe way too many sections.

Reply:
Yep, I thought of using `Server`, `HTTP_SERVER`or even `CherryPy`, but as you say it could lead us to have too many sections. `Server`can be used for more settings in the future.